test/ComposerRequireCheckerTest/FileLocator/LocateComposerPackageSourceFilesTest.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace ComposerRequireCheckerTest\FileLocator;
+
+use ComposerRequireChecker\FileLocator\LocateComposerPackageSourceFiles;
+use org\bovigo\vfs\vfsStream;
+use org\bovigo\vfs\vfsStreamDirectory;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @todo We're missing tests for the "classmap" and "files" keys.
```

Comment:
They didn't work for me out of the gate, so I left them out. I don't know if my tests had errors of if the code itself is buggy.